sample/src/main/AndroidManifest.xml

```diff
@@ -8,9 +8,10 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:allowNativeHeapPointerTagging="false"
```

Comment:
I'm not sure that we even need this sample app, but if we do, do we still need this `allowNativeHeapPointerTagging` attribute?

Reply:
This was removed in readdle latest changes